@xen-orchestra/lite/src/stories/web-core/ui/task-list/ui-task-list.story.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <ComponentStory v-slot="{ settings }" :params="[setting('tasks').widget(object()).preset(tasks)]">
+    <UiTaskList :tasks="settings.tasks" />
```

Comment:
```suggestion
  <ComponentStory v-slot="{ properties }" :params="[prop('tasks').type('Task[]').required().preset(tasks)]">
    <UiTaskList v-bind="properties" />
```
You can also add the markdown file to add details about the `Task` type.